lib/IMAP/FolderMapper.php

```diff
@@ -224,8 +224,7 @@ protected function guessSpecialUse(Folder $folder): void {
 			'junk' => ['junk', 'spam', 'bulk mail'],
 		];
 
-		$lowercaseExplode = explode($folder->getDelimiter(), $folder->getMailbox(), 2);
```

Comment:
I checked current `main` and that code is no longer there :+1:

Reply:
https://github.com/nextcloud/mail/blob/ec576848a9f0d3527720232f611f36107334e9c2/lib/IMAP/FolderMapper.php#L245